python/delta/tables.py

```diff
@@ -358,7 +360,13 @@ def forName(
         cls, sparkSession: SparkSession, tableOrViewName: str
     ) -> "DeltaTable":
         """
-        Create a DeltaTable using the given table or view name using the given SparkSession.
+        Instantiate a [[DeltaTable]] object using the given table or view name, If the given
+        tableOrViewName is invalid (i.e. either no table exists or an existing table is not a
+        Delta table), it throws a `not a Delta table` error.
+
+        The given tableOrViewName can also be the absolute path of a delta datasource(i.e.
```

Comment:
same nits as above

Reply:
Thanks for review, updated, please take a look again when you have a time.